redbot/core/core_commands.py

```diff
@@ -3633,7 +3633,19 @@ async def _set_api(
         else:
             if ctx.bot_permissions.manage_messages:
                 await ctx.message.delete()
+            embed = discord.Embed()
+            for dict_key, token in tokens.items():
+                if token.startswith("<") and token.endswith(">"):
+                    angle_bracket_warning = _(
+                        "Your `{dict_key}` was set including the <> symbols. If this was an accident you may experience"
+                        " issues with `{service}`. In that case, please try setting your `{dict_key}` again with"
+                        " the < and > removed"
+                    ).format(dict_key=dict_key, service=service)
+                    log.warning(angle_bracket_warning)
+                    embed.add_field(name="Warning", value=angle_bracket_warning)
             await ctx.bot.set_shared_api_tokens(service, **tokens)
+            if embed:
```

Comment:
This will be always the case with this command so this check is obsolete

You merge the card.send to send both the string and the new embed by specifying keyword arguments in that function